docs/Installation/Installation_Guide.rst

```diff
@@ -178,21 +173,22 @@ To create an inventory:
      node-03
      node-04
      node-05
-   
+
      [kube-node]
      node-01
      node-02
      node-03
      node-04
      node-05
-   
+
      [k8s-cluster:children]
      kube-node
      kube-master
 
-   Change the host names, IP addresses, and user names to conform to your infrastructure.
-   For example, if your servers are named "server1", "server2", and "server3", copy the code block
-   above and replace ALL instances of "node-0" with "server".
+   Change the host names, IP addresses, and user names to conform to your
+   infrastructure. For example, if your servers are named "server1", "server2",
+   and "server3", copy the code block above and replace ALL instances of

```

Comment:
Since the code block above seems to be made for a 5 server deployment, maybe the example should also mention up to "server5", or the example should be reduced to 3 servers.

Reply:
fixed.